docs/morgan.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[morgan]]
+== ECS Logging with Morgan
+
+This Node.js package provides a formatter for the https://github.com/expressjs/morgan#readme[morgan]
+logging middleware -- commonly used with Express -- compatible with
+{ecs-logging-ref}/intro.html[ELastic Common Schema (ECS) logging].
+In combination with the https://www.elastic.co/beats/filebeat[Filebeat] shipper,
+you can https://www.elastic.co/log-monitoring[monitor all your logs] in one
+place in the Elastic Stack.
+
+
+[float]
+=== Setup
+
+[float]
+[[morgan-setup-step-1]]
+=== Step 1: Install
+
+[source,cmd]
+----
+$ npm install @elastic/ecs-morgan-format
+----
+
+[float]
+[[morgan-setup-step-2]]
+=== Step 2: Configure
+
+[source,js]
+----
+const app = require('express')()
+const morgan = require('morgan')
+const ecsFormat = require('@elastic/ecs-morgan-format')
+
+app.use(morgan(ecsFormat())) <1>
+
+app.get('/', function (req, res) {
+  res.send('hello, world!')
+})
+
+app.listen(3000)
+----
+<1> Pass the ECS formatter to `morgan()`.
+
+See usage discussion and examples below.
+
+[float]
+[[morgan-setup-step-3]]
+==== Step 3: Configure Filebeat
+
+The best way to collect the logs once they are ECS-formatted is with https://www.elastic.co/beats/filebeat[Filebeat]:
```

Comment:
Here, I think it makes more sense to link to the docs than the product page.

```suggestion
The best way to collect the logs once they are ECS-formatted is with {filebeat-ref}[Filebeat]:
```

Reply:
Sounds good to me. FWIW, I copied this from https://github.com/elastic/ecs-logging-python/blob/master/docs/setup.asciidoc#install-filebeat